### PR TITLE
Revamp docs for Faster RCNN

### PR DESCRIPTION
**Pull request by abhi-glitchhg**
docs for faster rcnn 
part of #5897 

cc @NicolasHug

### CODE REVIEW

**Comment by abhi-glitchhg** [comment]
rendered docs are [here](https://output.circle-artifacts.com/output/job/caaddcbd-e632-4630-8f49-71bfad0fc73e/artifacts/0/docs/models/faster_rcnn.html)

File: torchvision/models/detection/faster_rcnn.py [hunk 1]
```diff
@@ -728,13 +767,26 @@ def fasterrcnn_mobilenet_v3_large_fpn(
...
+            download to stderr. Default is True.
         num_classes (int, optional): number of output classes of the model (including the background)
-        weights_backbone (MobileNet_V3_Large_Weights, optional): The pretrained weights for the backbone
-        trainable_backbone_layers (int, optional): number of trainable (not frozen) layers starting from final block.
-            Valid values are between 0 and 6, with 6 meaning all backbone layers are trainable. If ``None`` is
-            passed (the default) this value is set to 3.
+        weights_backbone (:class:`~torchvision.models.MobileNet_V3_Large_Weights`, optional): The
+            pretrained weights for the backbone.
+        trainable_backbone_layers (int, optional): number of trainable (not frozen) layers starting from
+            final block. Valid values are between 0 and 5, with 5 meaning all backbone layers are
```
Comment (datumbox) [inline torchvision/models/detection/faster_rcnn.py:781]:
```suggestion
            final block. Valid values are between 0 and 6, with 6 meaning all backbone layers are
```

File: torchvision/models/detection/faster_rcnn.py [hunk 2]
```diff
@@ -671,13 +697,26 @@ def fasterrcnn_mobilenet_v3_large_320_fpn(
...
```
Comment (datumbox) [inline torchvision/models/detection/faster_rcnn.py:711]:
```suggestion
            final block. Valid values are between 0 and 6, with 6 meaning all backbone layers are
```

**Review by datumbox — COMMENTED** [review]
Thanks @abhi-glitchhg! Some copy paste typos below:

**Comment by abhi-glitchhg** [comment]
Thanks @datumbox !

**Review by NicolasHug — APPROVED** [review]
Looks great, thanks @abhi-glitchhg !

**Pull request merged**